server.py

```diff
@@ -182,4 +182,11 @@ def file_upload(code, number):
         return "Solved! Great Job! "
 
 
+@app.error(404)
+def error404(error):
+    print(dir(error))
+    print(error.status_code)
+    return template("error404.html" ,errorcode=error.status_code , errorbody = error.body)
```

Comment:
print statements from debugging need to be removed.

Reply:
rename error404.html to error.html as it is now the common template for all errors

Reply:
i have all the changes suggested by you